src/include/rbd/librbd.h

```diff
@@ -81,7 +81,7 @@ typedef struct {
 } rbd_child_info_t;
 
 #define RBD_MAX_IMAGE_NAME_SIZE 96
-#define RBD_MAX_BLOCK_NAME_SIZE 24
+#define RBD_MAX_BLOCK_NAME_PREFIX_LENGTH 43
```

Comment:
Cannot change the ABI of the public API -- that is why I added a new method and deprecated ```block_name_prefix``` in the ```rbd_image_info_t``` struct. Instead, you could switch the Python bindings to issue a ```get_block_name_prefix``` API call when performing a stat.

Reply:
Oh, sorry, I almost forgot we couldn't change the public API. Thanks.